go.mod

```diff
@@ -139,4 +142,6 @@ replace (
 	// Fixes github.com/elazarl/goproxy Denial of Service (DoS)
 	// This dependency was removed from apimachinery 0.27.0
 	k8s.io/apimachinery => k8s.io/apimachinery v0.27.0
+	// Update knative/serving to avoid CVE-2023-48713
+	knative.dev/serving => knative.dev/serving v0.39.0
```

Comment:
Regarding the CVE-2023-48713 I seeing this report:
https://security.snyk.io/vuln/SNYK-GOLANG-KNATIVEDEVSERVINGPKGAUTOSCALERMETRICS-6091906

says 1.10.5 or 1.11.3 +

Reply:
actually, it seems that it should be 0.39.3:
https://github.com/opendatahub-io/kserve/pull/171

Reply:
The Dependabot alert says:

> Upgrade `knative.dev/serving` to version `0.39.0` or later

and ...

> The vulnerability has been patched in `v1.10.5`, `v1.11.3` and `v1.12.0`


However, the fix:
- https://github.com/knative/serving/pull/14523

... was back-ported to many more release streams, i.e. `v0.37.5` and `v0.38.3`:
- https://github.com/knative/serving/tree/v0.37.5/pkg/autoscaler/metrics
- https://github.com/knative/serving/tree/v0.38.3/pkg/autoscaler/metrics